app/controllers/sessions_controller.rb

```diff
@@ -8,9 +10,12 @@ def create
 
     token = api(:post, '/account/getToken', payload: payload)
 
-    session[:token] = token
-
-    redirect_to root_url, notice: "Logged in!"
+    if verify_recaptchas && token.present?
```

Comment:
On login the recaptcha v3 fails

![image](https://user-images.githubusercontent.com/8007722/72157790-c14e7480-3386-11ea-82bb-ca15d93a8931.png)